### ISSUE TITLE
Style Book: Several filter blocks show an error

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#8178]
## To reproduce

0. With WordPress 6.2 (or a previous version with GB installed) and a block theme like [Twenty Twenty-Three](https://wordpress.org/themes/twentytwentythree/) enabled.
1. Go to Appearance > Editor and edit a template.
2. In the toolbar, select Styles (black and white circle) and then, Open Style Book (eye icon).
3. Go to the WooCommerce tab.
4. Notice the Filter by Price, Filter by Rating and Filter by Stock blocks show an error instead of the real block.

![Filter by Price Controls block](https://user-images.githubusercontent.com/3616980/212304153-417f69c6-633c-4b23-af5f-73b4ca6101fe.png)

## Expected behavior

The block should be displayed properly.

**Comment by nefeline** [comment]
These filters were not loading as a consequence of the `“Uncaught RangeError: Maximum call stack size exceeded”` error related to the Mini Cart Contents Block (#8177), caused by the `useForcedLayout` hook: more specifically, [`replaceInnerBlocks`](https://github.com/woocommerce/woocommerce-blocks/blob/6f637dbbd9a00c2c911c73d6298d232fe7115f23/assets/js/blocks/cart-checkout-shared/use-forced-layout.ts#L134) ). With this error out of the way, all filter blocks are correctly rendered, but with one caveat: the older version is still displayed (related to issue reported on #8175 ).

In other words: fixing https://github.com/woocommerce/woocommerce-blocks/issues/8177 should also fix the error reported here for all filter blocks, allowing them to be previewed, but still in the old version (correct preview to be displayed as soon as https://github.com/woocommerce/woocommerce-blocks/issues/8175 is fixed).

Screenshot:

<img width="1216" alt="Screenshot 2023-01-23 at 02 15 50" src="https://user-images.githubusercontent.com/15730971/213952131-61f46f4c-9fd9-4929-8414-f46a6c500de3.png">

**Comment by Aljullu** [comment]
Thanks for your research, @nefeline! It looks like this one will be fixed with #8243. :slightly_smiling_face:
